plugins/poll/assets/javascripts/discourse/components/modal/poll-ui-builder.js

```diff
@@ -19,46 +18,26 @@ const VOTE_POLL_RESULT = "on_vote";
 const CLOSED_POLL_RESULT = "on_close";
 const STAFF_POLL_RESULT = "staff_only";
 
-export default class PollUiBuilderController extends Controller.extend(
-  ModalFunctionality
-) {
+export default class PollUiBuilderController extends Component {
   showAdvanced = false;
   pollType = REGULAR_POLL_TYPE;
-  pollTitle = "";
-  pollOptions = null;
-  pollOptionsText = null;
+  pollTitle = null;
```

Comment:
Seeing this is `undefined` by default, can we just do"

```suggestion
  pollTitle;
```